src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/MatchType.cs

```diff
@@ -5,8 +5,8 @@ namespace Microsoft.AspNetCore.Rewrite.Internal.IISUrlRewrite
 {
     public enum MatchType
     {
-        Pattern,
-        IsFile,
-        IsDirectory
+        pattern,
```

Comment:
Yeah, it didn't feel right writing it that way but I though that regardless of the parsing options it would be better for the enums to be consistent with the value casing from the Url Rewrite documentation